Sources/SwiftInspectorVisitors/PropertyVisitor.swift

```diff
@@ -51,6 +51,26 @@ public final class PropertyVisitor: SyntaxVisitor {
     return .skipChildren
   }
 
+  public override func visit(_ node: ClassDeclSyntax) -> SyntaxVisitorContinueKind {
```

Comment:
Should we also assert here to keep it consistent with other types?

Reply:
We don't need to assert – it's valid for this visitor to encounter decls. I only assert when we encounter a decl that wouldn't compile, like seeing any type decl within a protocol's `members` node.

I'm not convinced that asserting is actually a useful thing to do there since it's reasonable to run `SwiftInspector` over code that doesn't compile. But we can tackle that later